packages/react-app/src/constants.js

```diff
@@ -9,8 +9,10 @@ export const BLOCKNATIVE_DAPPID = "0b58206a-f3c0-4701-a62f-73c7243e8c77";
 
 export const ALCHEMY_KEY = "oKxs-03sij-U_N0iOlrSsZFr29-IqbuF";
 
-// RPC_POLL_TIME as 5 seconds to slower the rpc calls at start
-export const RPC_POLL_TIME = 5000;
+// LOCAL_RPC_POLL_TIME as 5 seconds to slower the rpc calls at start
+export const LOCAL_RPC_POLL_TIME = 5000;
+
+export const MAINNET_RPC_POLL_TIME = 30000;
```

Comment:
Also: Does everything works correctly if we set it to 0? My expectation would be that if 0 => every thing work "onBlock". Is that the case?

Reply:
Tried setting both `LOCAL_RPC_POLL_TIME ` and `MAINNET_RPC_POLL_TIME ` to 0 and it seems to work correctly, Yup if pollTime is passed as zero it defaults to `onBlock`. It's mentioned in eth-hooks docs also to be sure tried it out locally.

Currently if you are on `localhost` the poll time is passed as zero, So I tried mining an empty block to check it makes `eth_call` for `purpose` variable. Here is the test [repo](https://github.com/technophile-04/scaffold-eth/tree/rpc-calls-sensible-defaults-test) for it
![Screenshot 2022-09-28 at 4 42 35 PM](https://user-images.githubusercontent.com/80153681/192782751-5aa7bc31-c9e4-4919-ba31-b87f4b3caf02.jpg)